website/sidebars.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
         "concepts/provider_observer",
         "concepts/scopes",
         "concepts/provider_lifecycles",
-        "concepts/why_immutability",
+        "concepts/immutability",
```

Comment:
I think it's worth moving this next to "About codegen/hooks."

We could have an "About immutability"

Reply:
I'm not sure I agree with this, but I do think you're ultimately bringing up a good point. Currently, Immutability is underneath Concepts, and I think Immutability is a concept, more or less. However, I think your point is valid in that the organization of the pages for Riverpod documentation could use a little bit of restructuring.

For example, there's a section called All Providers, but there's also a page called "Providers". Granted, the Providers page is under Concepts, but if I recall correctly, this confused me several times when I first started navigating the Riverpod documentation. I kept going to the "Providers" page to look for documentation of the various the providers, which is under the All Providers section.

Perhaps, we can leave the above change as is for now and create a new issue to address the structure/organization/naming of pages/sections within the Riverpod documentation. What are your thoughts around this?

Reply:
The "concepts" bit is bound to disappear anyway. It's leftover from the old docs.
The sidebar will change significantly as the docs are updated.

Its correct place should be next to the "About codegen/hooks". They serve kind of the same purpose. It's one of the early design decisions people have to take when using Riverpod.
In fact, we could argue that the website could have a third toggle for enabling/disabling immutability

Those "About x" may move more up/down as the sidebar evolve, but they most definitely should be bundled with each other

Reply:
Sounds good to me, I'll get it moved.